app/src/main/java/com/kickstarter/ui/activities/CommentsActivity.kt

```diff
@@ -74,12 +74,15 @@ class CommentsActivity :
             .subscribe {
                 adapter.insertData(it, 0)
                 binding.commentsRecyclerView.scrollToPosition(0)
+                viewModel.inputs.postCommentToServer(it)
```

Comment:
this logic can be moved to the ViewHolderViewModel `viewModel.inputs.postCommentToServer(it)`